test/more-less/more-less.html

```diff
@@ -0,0 +1,102 @@
+
+					<p>Mauris in libero cursus, iaculis sapien quis, semper felis. Maecenas convallis gravida libero euismod vehicula. Morbi quis lectus dui. Praesent non congue purus, at vehicula metus. Quisque vitae tempus elit. Aenean a aliquet nunc, nec blandit metus. Duis mattis odio vel erat eleifend volutpat. Cras eget augue et ligula vehicula ultrices. Ut vitae hendrerit nibh, id interdum nisl. Ut vestibulum tellus sed nulla bibendum aliquam. Maecenas quis sapien enim.</p>
+				</d2l-more-less>
+			</template>
+		</test-fixture>
+		<test-fixture id="expanded-dynamic">
+			<template>
+				<d2l-more-less expanded>
+					<p id="clone-target">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum elementum venenatis arcu sit amet varius. Maecenas posuere magna arcu, quis maximus odio fringilla ac. Integer ligula lorem, faucibus sit amet cursus vel, pellentesque a justo. Aliquam urna metus, molestie at tempor eget, vestibulum a purus. Donec aliquet rutrum mi. Duis ornare congue tempor. Nullam sed massa fermentum, tincidunt leo eu, vestibulum orci. Sed ultrices est in lacus venenatis, posuere suscipit arcu scelerisque. In aliquam ipsum rhoncus, lobortis ligula ut, molestie orci. Proin scelerisque tempor posuere. Phasellus consequat, lorem quis hendrerit tempor, sem lectus sagittis nunc, in tristique dui arcu non arcu. Nunc aliquam nisi et sapien commodo lacinia. <a href>Quisque</a> iaculis orci vel odio varius porta. Fusce tincidunt dolor enim, vitae sollicitudin purus suscipit eu.</p>
+				</d2l-more-less>
+			</template>
+		</test-fixture>
+
+		<script type="module">
+			import { runAxe } from '../../test-helpers/a11y-helper.js';
+
+			describe('more-less', () => {
+				let moreless, content;
+
+				function waitForInitialization(callback) {
+					if (content.style.height === '') {
+						setTimeout(() => { waitForInitialization(callback);}, 10);
+					} else {
+						setTimeout(() => { // Need the second timeout here to give the transition a chance to finish
+							callback();
+						}, 400);
+					}
+				}
+
+				describe('basic', () => {
+					beforeEach(async() => {
+						moreless = fixture('basic');
+						await moreless.updateComplete;
+						content = moreless.shadowRoot.querySelector('.more-less-content');
+						await new Promise(resolve => {
+							waitForInitialization(resolve);
+						});
+					});
+					it('should pass all axe tests', async() => {
+						await runAxe(moreless);
+					});
+				});
+
+				describe('expanded', () => {
+					beforeEach(async() => {
+						moreless = fixture('expanded-dynamic');
+						await moreless.updateComplete;
+						content = moreless.shadowRoot.querySelector('.more-less-content');
+						await new Promise(resolve => {
+							waitForInitialization(resolve);
+						});
+					});
+					it('should pass all axe tests when expanded', async() => {
+						await runAxe(moreless);
+					});
+					it('should expand when more content is dynamically added', async() => {
```

Comment:
This test can be checked with perceptual diff if a specific test like this is written, but it's caught a lot of regressions in the past so I ported it over for now.  We can remove it once a perceptual diff test exists.  All other tests seemed like they could be tested with perceptual diff: https://github.com/BrightspaceUI/more-less/blob/master/test/d2l-more-less.html (what it looks like closed, open, and hidden because the text is too short)